### DISCUSSION TITLE
showDialog Example

### DISCUSSION THREAD

**Discussion by omermahmood** [EnsembleUI/ensemble#295] [General]
Where can I find documentation for showDialog? I started working on a sample app to learn, and want to show a dialog on app load. I could not find the documentation, and I am not getting any error message.

````
View:
  title: Todo
  onLoad:
    action: showDialog("hello")

  Column:
    styles: { mainAxis: center }
    children:
      - Row:
          styles: { mainAxis: center }
          children:
            - Text:
                text: Hello world!
                textStyle: italic`

**Comment by vusters** [comment]
@omermahmood our Kitchen Sink is the best place to learn about Ensemble. Here is an example of how to use showDialog action https://studio.ensembleui.com/app/e24402cb-75e2-404c-866c-29e6c3dd7992/screen/HRIajN2i8bDDVTL32j8m

**Reply by omermahmood** [reply]
Thank you @vusters 
Is there a way to search for it in docs or kitchen sink?
When I search for `showDialog` in Docs, it did not return any result.

**Reply by kmahmood74** [reply]
yes our documentation is lacking. You can help. This is our doc repo - https://github.com/EnsembleUI/ensemble_docs Feel free to make changes and additions and create PRs. It will be much appreciated
